### PR TITLE
User register2

### PR DESCRIPTION
**Pull request by robimaliqi**

### CODE REVIEW

File: client/src/pages/register/Register.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import { Header } from "../../components/Header/Header";
```
Comment (Hafizalimi7) [inline client/src/pages/register/Register.js:1]: nice, work

File: client/src/pages/register/Register.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { getByTestId, render, screen } from "@testing-library/react";
```
Comment (Hafizalimi7) [inline client/src/pages/register/Register.test.js:1]: good code , easy to read

**Review by Hafizalimi7 — APPROVED** [review]

File: client/package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```
Comment (sophiegilder) [inline client/package.json:9]: I think we're using fetch elsewhere? Do we need axios too?

**Review by sophiegilder — CHANGES_REQUESTED** [review]
I don't think we need axios?

File: client/package.json (hunk 3 above)
Comment (robimaliqi) [reply client/package.json:9]: no we don't need it, I was doing another tutorial and needed it fo that but I don't think it's going to effect anything with it being there?

**Review by sophiegilder — APPROVED** [review]

**Pull request merged**